lib/tasks/create_admin.rake

```diff
@@ -0,0 +1,15 @@
+namespace :admin do
```

Comment:
Good Work 👍 

While it's good that you know how to create a rake task and that rake task is nothing but just another process to execute your ruby code, this one is not the exact use case.

We generally seed data via seed file or rails console.So please remove this task.